frontend-cert/js-projects/inventory-management-app/script.js

```diff
@@ -0,0 +1,67 @@
+let inventory = [];
+
+const findProductIndex = (productName) => {
```

Comment:
It doesn't really matter how users find the index, but I'm curious if they will know the `indexOf` method at this point?

Reply:
I guess they should but I didn't find a reference in the sheet. It should be taught in some workshop. In that case do you think I should change the user stories and ask to use `findIndex` instead?

Reply:
At this point, `indexOf` has been introduced according to the sheet.

Reply:
`indexOf` does not work here because we need to search for the object key inside an array of objects.